packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/native/emptyTemplateTs/src/components/__tests__/HelloWorld.spec.tsx.ejs

```diff
@@ -2,14 +2,14 @@ import { createElement } from "react";
 import { shallow, ShallowWrapper } from "enzyme";
 import { Platform } from "react-native";
 
-import { HelloWorldSample, HelloWorldSampleProps } from "../HelloWorldSample";
+import { HelloWorld, HelloWorldProps } from "../HelloWorld";
 
-describe("HelloWorldSample", () => {
-    const render = (props: HelloWorldSampleProps): ShallowWrapper => shallow(<HelloWorldSample {...props} />);
+describe("HelloWorld", () => {
+    const render = (props: HelloWorldProps): ShallowWrapper => shallow(<HelloWorld {...props} />);
```

Comment:
I think it is better to get rid of such functions - they don't add much

Reply:
Removed them from native templates, web will follow in another PR.